api/src/main/java/org/openmrs/module/m2sysbiometrics/bioplugin/AbstractBioServerClient.java

```diff
@@ -113,17 +109,12 @@ public String delete(String subjectId) {
     }
 
     @Override
-    public boolean isFingerScanExists(M2SysCaptureResponse scannedFingers) {
-        String response = identify(scannedFingers.getTemplateData());
-        M2SysResults results = XmlResultUtil.parse(response);
-        return CollectionUtils.isNotEmpty(results.toOpenMrsMatchList());
+    public boolean isServerUrlConfigured() {
+        String propertyValue = adminService.getGlobalProperty(getServerUrlPropertyName());
```

Comment:
can use the getServiceUrl method here.

Reply:
getServiceUrl method will throw an exception if service url is not defined/blank: according to your comment above, in order to avoid quering the national server, it is needed to check if national url configured

Reply:
ok, leave it as it then